src/revwalk.c

```diff
@@ -38,7 +38,7 @@ git_commit_list_node *git_revwalk__commit_lookup(
 	return commit;
 }
 
-static int push_commit(git_revwalk *walk, const git_oid *oid, int uninteresting, int from_glob)
+int git_revwalk__schedule_commit(git_revwalk *walk, const git_oid *oid, const git_revwalk__schedule_options *opts)
```

Comment:
Hum. I'm definitely happy to expose these functions internally. But even though I'd prefer `schedule` in hindsight, we are already set in stone because of `git_revwalk_push` is exposed to users of libgit2 already. And mixing `schedule` and `push` if both in fact do the same thing is going to confuse people :/ So unfortunately, I'd prefer to stick with `git_revwalk__push_*`.

Reply:
Agreed, it doesn't feel that useful to warrant a change, so let's wait for a "bigger impact change" (say, parallel revwalking) to revisit the public API/deprecate things. I've reverted to the current "push" naming.